README.md

```diff
@@ -14,14 +17,58 @@ See `defaults/main.yml`
 ## Example Playbook
 
 ```yaml
-  - hosts: servers
-    roles:
-    - role: marvel-nccr.bigdft
+- hosts: servers
+  roles:
+  - role: marvel-nccr.bigdft
+```
+
+## Development and testing
+
+This role uses [Molecule](https://molecule.readthedocs.io/en/latest/#) and [Docker](https://www.docker.com/) for tests.
+
+After installing [Docker](https://www.docker.com/):
+
+Clone the repository into a package named `marvel-nccr.bigdft` (the folder must be named the same as the Ansible Galaxy name)
+
+```bash
+git clone https://github.com/marvel-nccr/ansible-role-bigdft marvel-nccr.bigdft
+cd marvel-nccr.bigdft
+```
+
+Then run:
+
+```bash
+pip install -r requirements.txt  # Installs molecule
+molecule test  # runs tests
 ```
 
+or use tox (see `tox.ini`):
+
+```bash
+pip install tox
+tox
+```
+
+## Code style
+
+Code style is formatted and linted with [pre-commit](https://pre-commit.com/).
+
+```bash
+pip install pre-commit
+pre-commit run -all
+```
+
+## Deployment
+
+Deployment to Ansible Galaxy is automated *via* GitHub Actions.
+Simply tag a release `vX.Y.Z` to initiate the CI and release workflow.
+Note, the release will only complete if the CI tests pass.
```

Comment:
sweet! Once it passes, I'll release a new version using the new deployment automation 😄

Reply:
it just finished locally .. Do I put 20.04 back in this branch ?

Reply:
nah lets handle that in a separate PR (just open it once this is merged). I've got no patience left to wait for another build 😆

Reply:
then I can get it in to quantum mobile (which is 18.04 anyway) and start that build